### PR TITLE
Feature | WDF Extras

### PR DESCRIPTION
**Pull request by ChazUK**

### CODE REVIEW

**Comment by ChazUK** [comment]
[(5) WDF - Update Workplace information](https://trello.com/c/v8AAlPE8/63-5-wdf-update-workplace-information)

**Comment by ChazUK** [comment]
[(5) WDF - Update Worker Record](https://trello.com/c/W7ZHJv93/62-5-wdf-update-worker-record)

File: src/app/core/model/wdf.model.ts [hunk 1]
```diff
@@ -0,0 +1,4 @@
+export interface WDFValue {
+  isEligible: 'Yes' | 'No' | 'Not relevant';
```
Comment (omarCreativeDev) [inline src/app/core/model/wdf.model.ts]: is this better placed being an enum to avoid string matching?

Comment (ChazUK) [reply src/app/core/model/wdf.model.ts]: done

**Pull request merged**